addons/restrictSites-mpd-addon.php

```diff
@@ -322,6 +322,7 @@ function mpd_is_site_active(){
 function change_tax_terms_table(){
 
 	$options 		= get_option( 'mdp_settings' );
+	if ( isset( $options[ 'master_site_setting' ] ) ) {
```

Comment:
This fix is not sufficient. The `$master_id` var can be undefined within the if statement below due to this change.
If statement should also check if the var exists or a better solution in my opinion is to set the var with a null value and check if there is a value in the if statement.